app/src/main/java/com/github/onlynotesswent/model/folder/FolderViewModel.kt

```diff
@@ -430,6 +433,28 @@ class FolderViewModel(private val repository: FolderRepository) : ViewModel() {
         onFailure = onFailure)
   }
 
+  /**
+   * Retrieves all friends only folders from a list of following users. If the list is null, an
+   * empty list is used.
+   *
+   * @param followingListIds The list of users IDs to retrieve friends only folders from.
+   * @param onSuccess The function to call when the friends folders are retrieved successfully.
+   * @param onFailure The function to call when the friends folders fail to be retrieved.
+   */
+  fun getFoldersFromFollowingList(
+      followingListIds: List<String>?,
+      onSuccess: () -> Unit = {},
```

Comment:
These functions do not have the List<Folder> since they do not return the list of folders but update their respective states. Let me know what do you, since we do not really need this parameter but it is true that it is not consistent

Reply:
> they do not return the list of folders but update their respective states

We could say the same for notes and decks then and with other functions as well. I think it would be best to stay consistent with the previous implementation